settings.py

```diff
@@ -22,3 +22,8 @@
 
 # Prefix this NApp has when using cookies
 COOKIE_PREFIX = 0xAA
+
+# Maximum number of paths to consider when calculating the disjoint paths
+# i.e., the number of paths that will be requested to pathfinder to calculate
+# the maximum disjoint paths from unwanted_path
+DISJOINT_PATH_CUTOFF = 10
```

Comment:
This value was chosen based on the experiments performed on the production network topology: up to 7 paths were necessary from pathfinder to get a disjoint path from the best path on each and every pair of possible UNIs. Using 10 gives a safety margin to also accommodate some future changes on the topology (of course  we can still review it later)